### ISSUE TITLE
Add note about difference between Mautic 2 and Mautic 3 Mautibox

### ISSUE THREAD

**Issue by dennisameling** [mautibox/mautibox#7]
We currently have 2 domains:
- mautibox.com
- m3.mautibox.com

This causes confusions for some users.
Please, if possible, add a note to both domains:

- mautibox.com: "This version of Mautibox only works for Pull Requests that are created for Mautic 2. Looking to test Mautic 3 PRs? Please go to https://m3.mautibox.com"
- m3.mautibox.com: "This version of Mautibox only works for Pull Requests that are created for Mautic 3. Looking to test Mautic 2 PRs? Please go to https://mautibox.com"

**Comment by adiwegs** [comment]
Yes that would be perfect. If you like you could make it even simpler:

Looking to test Mautic 3 Pull Requests? [Test Mautic 3]

**Comment by heathdutton** [comment]
Sorry guys, just now seeing this and you are right. I've been holding off making changes because I'm hoping we'll transition to M3 only soon and merge these to one. What's the timeline like on that, do you think?

**Comment by dennisameling** [comment]
After 3.x is released (let's hope some time in May), we're planning to support the 2.16 branch (security/critical bug fixes only) at least for a few months to allow users enough time to upgrade. So there will be PRs coming in for the 2.16 branch until around the end of this year I think. However, I expect the amount of PRs for 2.16 to be rather low after the 3.x release, so I'm not sure whether it's worth it to keep a 2nd Mautibox version live just for that. We could test those PRs locally if that's easier.

Previously, version statistics were available at https://mautic.org/stats, but that page is (temporarily) gone. That'd allow us to monitor 2.x usage so that it's easier to decide when we stop supporting 2.16.

I'm happy to provide a PR for the home page, but I only came across https://github.com/mautibox/mautibox/blob/master/web/index.php which is for 3.x. I'm also not seeing any branches in the repo, so if you could point me in the right direction where I can edit the 2.x Mautibox home page, I'll provide a PR 😄
